proto/api/v2/compliance_results_service.proto

```diff
@@ -76,6 +76,8 @@ message ComplianceScanStatsShim {
 message ComplianceProfileScanStats {
   repeated ComplianceCheckStatusCount check_stats = 1;
   string profile_name = 2;
+  string title = 3;
+  string version = 4;
```

Comment:
Is this a version in the sense of, for example `1.1.1`?
I see in the test fixtures the version is a string of `fmt.Sprintf("test_version_%s", profileName)`?

Reply:
this is a version like "1.1.1"
I just did that for testing purpose

Reply:
CO has profile versions such as:
`1.4.0`
`Revision 4`
`V1R1`

so it could really be anything.